src/expr/src/interpret.rs

```diff
@@ -648,8 +627,8 @@ impl Interpreter for Trace {
         // TODO: this is duplicative! If we have more than one or two special-cased functions
         // we should find some way to share the list between `Trace` and `ColumnSpecs`.
         let (left_monotonic, right_monotonic) = match func {
-            BinaryFunc::JsonbGetString { stringify: false } => (Monotonic::Yes, Monotonic::No),
-            _ => binary_monotonic(func),
+            BinaryFunc::JsonbGetString { stringify: false } => (true, false),
```

Comment:
nit: the comment explains that the code is duplicated, but not why the special-casing is needed here. Why can't this clause live under the `BinaryFunc::JsonbGetString` case of the `BinaryFunc::is_monotone()` definition?

Reply:
The variable naming here is wrong: these should be `left_pushdownable` etc. and not `left_monotonic`. (JsonbGetString is the one function we can push down right now that isn't monotonic... we have hardcoded support for it.)

I've fixed up the naming. I'm also in the process of adding the second such function -- `=` -- and I'll see if I can get rid of the duplication entirely there as well.

Reply:
Maybe we could have `is_pushdownable` function on the `~Func` types that special-cases `JsonbGetString` and otherwise just delegates to `is_monotonic`.